.ci/openshift-ci/test.sh

```diff
@@ -65,44 +66,44 @@ END
 }
 
 deployChe() {
-  chectl server:deploy  --che-operator-cr-patch-yaml=custom-resources.yaml --telemetry=off --platform=openshift --installer=operator --batch
+  chectl server:deploy --che-operator-cr-patch-yaml=custom-resources.yaml --telemetry=off --platform=openshift --installer=operator --batch
 }
 
 patchTestPodConfig(){
-  # obtain the basic "happy-path" pod config
-  cat .ci/openshift-ci/happy-path-pod.yaml > happy-path-pod.yaml
+  # obtain the basic test pod config
+  cat .ci/openshift-ci/plugins-test-pod.yaml > plugins-test-pod.yaml
 
-  # Patch the basic "happy-path" pod config
+  # Patch the basic test pod config
   ECLIPSE_CHE_URL=http://$(oc get route -n "eclipse-che" che -o jsonpath='{.status.ingress[0].host}')
-  sed -i "s@CHE_URL@${ECLIPSE_CHE_URL}@g" happy-path-pod.yaml
-  cat happy-path-pod.yaml
+  sed -i "s@CHE_URL@${ECLIPSE_CHE_URL}@g" plugins-test-pod.yaml
+  cat plugins-test-pod.yaml
 }
 
 runTest() {
   # Create the test pod
   oc create namespace $TEST_POD_NAMESPACE
-  oc apply -f happy-path-pod.yaml
+  oc apply -f plugins-test-pod.yaml
 
   # wait for the pod to start
   while true; do
     sleep 3
-    PHASE=$(oc get pod -n ${TEST_POD_NAMESPACE} happy-path-che \
+    PHASE=$(oc get pod -n ${TEST_POD_NAMESPACE} ${TEST_PATH_POD_NAME} \
         --template='{{ .status.phase }}')
     if [[ ${PHASE} == "Running" ]]; then
         break
     fi
   done
 
   # wait for the test to finish
-  oc logs -n ${TEST_POD_NAMESPACE} happy-path-che -c happy-path-test -f
+  oc logs -n ${TEST_POD_NAMESPACE} ${TEST_PATH_POD_NAME} -c plugins-test -f
 
   # just to sleep
   sleep 3
 
   # download the test results
   mkdir -p /tmp/e2e
-  oc rsync -n ${TEST_POD_NAMESPACE} ${HAPPY_PATH_POD_NAME}:/tmp/e2e/report/ /tmp/e2e -c download-reports
-  oc exec -n ${TEST_POD_NAMESPACE} ${HAPPY_PATH_POD_NAME} -c download-reports -- touch /tmp/done
+  oc rsync -n ${TEST_POD_NAMESPACE} ${TEST_PATH_POD_NAME}:/tmp/e2e/report/ /tmp/e2e -c download-reports
+  oc exec -n ${TEST_POD_NAMESPACE} ${TEST_PATH_POD_NAME} -c download-reports -- touch /tmp/done
```

Comment:
it looks like all this script could be in che e2e tests as the only 'plugin specific stuff' seems to be where we say: use 'test-all-plugins'

Reply:
I am not sure that this discussion is a topic of this issue.
If your point to remove this script from the "che-plugin-registry" repository,
please create specific issue for that and we can continue the discussion.

Reply:
well it should not have been merged like that, we're creating technical debt

Reply:
We can always abstract them out at a later time, right now we're trying to get the tests to run properly first